### ISSUE TITLE
Mysql timeout

### ISSUE THREAD

**Issue by f4ble** [f4ble/Arkon#5]
Despite my attempts to set up heartbeat and mysql timeout prevention it does seem to time out during the night with no activity. 

This should not be a big problem to solve. #todo

**Comment by f4ble** [comment]
Still trying to figure out how to stop these sql errors from happening. I tried manually stopping the mysql server after pyarc connected and successfully trapped for that error, but I keep getting 

ConnectionResetError: "An existing connection was forcibly closed by the remote host"
(Thrown by pymysql, not sqlalchemy)

Now this would normally be easily solved by increasing the timeout through an sql query, but this doesn't work for some reason or rather since the connection is persistent over several days it's not a good enough solution. I tried to recycle the connection, but sqlAlchemy is giving me trouble with this. I am slightly annoyed that sqlAlchemy doesn't restore the connection anyways... to be continued.
